issue_template.md

```diff
@@ -0,0 +1,31 @@
+#### brief description
```

Comment:
Could add a capital letter to your titles ?

Reply:
yes. done.